tests/rest/client/v1/test_rooms.py

```diff
@@ -49,10 +49,17 @@ def make_homeserver(self, reactor, clock):
             "red",
             http_client=None,
             federation_client=Mock(),
-            ratelimiter=NonCallableMock(spec_set=["can_do_action"]),
+            request_ratelimiter=NonCallableMock(
+                spec_set=["can_do_action", "ratelimit", "rate_hz", "burst_count"]
+            ),
+            login_ratelimiter=NonCallableMock(spec_set=["can_do_action", "ratelimit"]),
         )
-        self.ratelimiter = self.hs.get_ratelimiter()
-        self.ratelimiter.can_do_action.return_value = (True, 0)
+        self.request_ratelimiter = self.hs.get_request_ratelimiter()
+        self.request_ratelimiter.can_do_action.return_value = (True, 0)
+        self.request_ratelimiter.rate_hz = Mock()
```

Comment:
Seems weird we have to set `rate_hz` on this one, isn't that essentially internal?